app/services/open_library/response_adapter.rb

```diff
@@ -0,0 +1,33 @@
+module OpenLibrary
+  class ResponseAdapter
+    def initialize(body, isbn, status)
+      @body = body
+      @isbn = isbn
+      @status = status
+    end
+
+    def book_response
+      {
+        "isbn": @isbn,
+        "title": book_body['title'],
+        "subtitle": book_body['subtitle'],
+        "number_of_pages": book_body['number_of_pages'],
+        "authors": book_body['authors']
+      }
+    end
+
+    def success?
+      @status >= 200 && @status < 400 && @body.present?
+    end
+
+    def errors
+      { "errors": 'The requested resource was not found' }
```

Comment:
For this scope it's ok but ideally you want to be able to handle errors in a separate concern, to unify errors between services and controllers